app/features/navbar/styled/drawerContainer.js

```diff
@@ -0,0 +1,8 @@
+// @flow
```

Comment:
This file should be called DrawerContainer. I guess you didn't notice the problem because macOS uses a case-insensitive filesystem.

Reply:
Done.

Reply:
Did you force push? I don't see the change.

Reply:
Weird.
Git doesn't even consider this change in macOS. 😨
I updated it using Github website and squash generated commit.